examples/transit_vnet_common/versions.tf

```diff
@@ -11,5 +11,9 @@ terraform {
 }
 
 provider "azurerm" {
-  features {}
+  features {
+    resource_group {
+      prevent_deletion_if_contains_resources = false
```

Comment:
Shouldn't happen, but please verify the resource group will be retained if an existing resource group is used.

Reply:
@migara this happens only in scenarios where AppInsights is deployed. Two resources are typically left. They are not created by Terraform. 

I also thought that since this is an example, not production code, we should just cleanup after our selves. Maybe a comment about that would nice.

Reply:
> Shouldn't happen, but please verify the resource group will be retained if an existing resource group is used.

It will, this is used only when we create an RG with our code